### PR TITLE
chore: add relevant files to paths-ignore in workflow

### LINKED ISSUE #1081: Add relevant files to paths-ignore in workflow.
### Is your feature request related to a problem? Please describe.

Currently, github actions will run even when we make changes to files like `*.md`

### Describe the solution you'd like

Add these files under paths-ignore in github actions. 

### Additional context

https://stackoverflow.com/questions/59759921/how-to-skip-github-actions-job-on-push-event

### PR DESCRIPTION
**Pull request by KushalBeniwal**
### Description

Added md files to paths-ignore in github actions

Fixes #1081

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

Tested using existing tests by running
`python -m unittest discover tests`

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -8,6 +8,8 @@ on:
     branches: [develop]
```
Comment (epicadk) [inline .github/workflows/main.yml:8]: here as well and then it's good to go

**Review by epicadk — APPROVED** [review]

**Review by vj-codes — APPROVED** [review]
Thank you for your contribution @KushalBeniwal 
LGTM 🎉

**Comment by vj-codes** [comment]
Adding label ready to merge since no code has been changed

**Comment by isabelcosta** [comment]
thank you for your contribution @KushalBeniwal 🎉

**Pull request merged**